js/smart-side-nav.js

```diff
@@ -0,0 +1,56 @@
+namespacer('bcpl');
+
+bcpl.smartSideNav = (($, window) => {
+	const navLinksSelector = '.secondary-nav nav ul li a';
+	let activeWindow = window;
+
+	const getHrefWithoutQueryString = (href) => {
+		return href ? href.toLowerCase().split('?')[0] : undefined;
+	};
+
+	const compareNavLinks = (index, navLink) => {
+		const $navLink = $(navLink);
+
+		if (!$navLink.attr('href')) {
+			$navLink.removeClass('active');
+			return true;
+		}
+
+		const hrefWithoutQueryString = getHrefWithoutQueryString($navLink.attr('href'));
```

Comment:
since you use the ```$navLink.attr('href')``` potentially multiple times, i would store it in a variable.

Reply:
Good call. Done.